spec/ndx-microscopy.extensions.yaml

```diff
@@ -258,3 +258,69 @@ groups:
       doc: Link to VolumetricImagingSpace object containing metadata about the region of physical space this imaging data
         was recorded from.
       target_type: VolumetricImagingSpace
+
+  - neurodata_type_def: MultiChannelMicroscopyVolume
+    neurodata_type_inc: NWBData
+    doc: Static (not time-varying) volumetric imaging data acquired from multiple optical channels.
```

Comment:
@rly Any idea why this would cause

```python
src/pynwb/tests/test_constructors.py:5: in <module>
    from ndx_microscopy.testing import (
src/pynwb/ndx_microscopy/__init__.py:21: in <module>
    load_namespaces(str(__spec_path))
/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/hdmf/utils.py:672: in func_call
    return func(**pargs)
/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/pynwb/__init__.py:139: in load_namespaces
    return __TYPE_MAP.load_namespaces(namespace_path)
/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/hdmf/utils.py:668: in func_call
    return func(args[0], **pargs)
/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/hdmf/build/manager.py:482: in load_namespaces
    deps = self.__ns_catalog.load_namespaces(**kwargs)
/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/hdmf/utils.py:668: in func_call
    return func(args[0], **pargs)
/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/hdmf/spec/namespace.py:537: in load_namespaces
    ret[ns['name']] = self.__load_namespace(ns, reader, resolve=resolve)
/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/hdmf/spec/namespace.py:447: in __load_namespace
    self.__load_spec_file(reader, s['source'], catalog, types_to_load=types_to_load, resolve=resolve)
/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/hdmf/spec/namespace.py:401: in __load_spec_file
    temp_dict = {k: None for k in __reg_spec(self.__group_spec_cls, spec_dict)}
/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/hdmf/spec/namespace.py:[38](https://github.com/catalystneuro/ndx-microscopy/actions/runs/9649530312/job/26613211632?pr=13#step:9:39)7: in __reg_spec
    spec_obj = spec_cls.build_spec(spec_dict)
/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/hdmf/spec/spec.py:99: in build_spec
    return cls(**kwargs)
/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/hdmf/utils.py:667: in func_call
    pargs = _check_args(args, kwargs)
/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/hdmf/utils.py:660: in _check_args
    raise ExceptionType(msg)
E   TypeError: NWBGroupSpec.__init__: incorrect type for 'neurodata_type_inc' (got 'NWBDatasetSpec', expected 'NWBGroupSpec or str')
```

?

This occurs even if I use `NWBDataInterface`, which is what they use in https://github.com/focolab/ndx-multichannel-volume/blob/main/spec/ndx-multichannel-volume.extensions.yaml#L134-L135

Reply:
Nevermind @rly we fixed it; `NWBDataInterface` is what it was supposed to be, but I must have hallucinated its failure on my local device